controllers/cluster/api/v1/cluster_types.go

```diff
@@ -37,6 +37,27 @@ type ClusterSpec struct {
 type ClusterStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+	Status string `json:"status,omitempty"`
+}
+type Status int
+
+const (
+	Unknown Status = iota
+	Creating
+	Running
+)
+
+func (s Status) String() string {
+	switch s {
+	case Unknown:
+		return "Unknown"
+	case Creating:
+		return "Creating"
+	case Running:
+		return "Running"
+	default:
+		return "Error"
+	}
```

Comment:
Try Keep status name with k8s is better: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/